cmd/tfengine/README.md

```diff
@@ -91,70 +92,88 @@ Use our [example](../../examples/tfengine) configs to quickly get started.
 The engine takes a path to an input config and a path to output the generated
 Terraform configs. For details on fields for the input schema, see the
 [schema](../../internal/tfengine/schema.go). After the output has been generated,
-there is no dependency on the engine any longer, and the user can directly use
+there is no longer a dependency on the engine and the user can directly use
 the `terraform` and `terragrunt` binaries to deploy the infrastructure.
 
-Replace the values in the [example](../../examples/tfengine/simple.hcl) with
-values for your infrastructure, then run the following commands:
+1. Replace the values in the [example](../../examples/tfengine/simple.hcl) with
+   values for your infrastructure.
+
+1. To clone the repo, run the following commands:
+
+    ```shell
+    git clone https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite
+    cd healthcare-data-protection-suite
+    ```
+1. To set up helper environment vars, run the following commands:
+
+    ```shell
+    CONFIG_PATH=examples/tfengine/simple.yaml
+    OUTPUT_PATH=/tmp/engine
+    ```
+
+1. To install the engine, run the following command: 
+
+    ```shell
+    go install ./cmd/tfengine
+    ```
+
+1. To generate Terraform configs, edit the config with values for your
+   infra and then run the following commands:
 
-```shell
-# Step 1: Clone repo
-git clone https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite
-cd healthcare-data-protection-suite
+    ```shell
```

Comment:
Since we are saying to edit the config earlier on we can just trim this down to something like "Run the engine to generate your Terraform configs"

Reply:
Done, thanks